test/games/samus_returns/gui/dialog/test_msr_game_export_dialog.py

```diff
@@ -344,6 +344,12 @@ def test_get_game_export_params_custom(skip_qtbot, tmp_path):
 
 def test_export_button_without_remote(skip_qtbot, tmp_path, mocker):
     # Setup
+    mocker.patch("platform.system", return_value="Windows")
```

Comment:
This stuff is only necessary until #6770 is fixed, right?

Reply:
Yes